plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemRawNBT.java

```diff
@@ -291,27 +275,75 @@ else if (tag instanceof EndTag) {
         }
     }
 
-    ItemTag item;
+    @Override
+    public MapTag getPropertyValue() {
+        if (NMSHandler.getVersion().isAtMost(NMSVersion.v1_19)) {
```

Comment:
the purpose of the `raw_nbt` property is to persist any data that is not properly tracked by Denizen - I'm not sure it makes sense to disable the property on 1.20? At least not without a replacement

Reply:
Note this comment is on very specifically the getPropertyValue line, ie the part where it grabs specifically the subset of NBT keys that Denizen doesn't currently handle natively. This is not random/fake keys, this is just non-Denizen-supported keys.

For example `EntityTag`, `BucketVariantTag`, `Decorations`, Lodestone are all pre-documented vanilla tags that are intentionally tracked here because other Denizen item properties do not cover them properly. There's probably several more vanilla keys that Denizen lacks that aren't explicitly listed.

Any hacks based on the tags/mechs here are naturally a problem, but the property itself is essential until cleanly replaced, otherwise any item data not yet supported by other Denizen item properties will break.

Reply:
Confused the naming, sorry lol - meant `raw_components`, I.e. a property that has a white/blacklist of NMS component names, and specifically persists them in NBT format.

Already have a WIP locally which we can wait and include here if you want to, but didn't want to make the PR too big since it's all currently in unstable dev builds anyway

Reply:
If it's not in this PR, this PR needs to not kill data on items

Reply:
It (and most other NBT-related stuff) were/are already broken on 1.20.6 - since NBT structure is completely different `raw_nbt`s handling (and all other places using it to edit raw nbt) needs to be basically entirely reworked to work the way it used to on modern versions, which seems a bit silly to do when it's going to be replaced soon anyway.

I did make sure not to break older versions (It should still work fine on 1.19>), but I don't think keeping the 1.20.6 dev builds as unstable as they already are for a little longer is that bad/worth the trouble